modules/aggregation-layers/package.json

```diff
@@ -16,6 +16,7 @@
     "type": "git",
     "url": "https://github.com/visgl/deck.gl.git"
   },
+  "types": "dist/types/index.d.ts",
```

Comment:
I thought the plan for 8.8 was to generate types but not expose them via the "types" field?

Reply:
Yes I'll need to submit a separate PR for that. For now, this allows us to test the generated types internally.